python_modules/libraries/dagster-dg/dagster_dg/cache.py

```diff
@@ -47,6 +47,12 @@ def __init__(self, root_path: Path, logging_enabled: bool):
         self._root_path.mkdir(parents=True, exist_ok=True)
         self._logging_enabled = logging_enabled
 
+    def delete(self, key: Tuple[str, ...]) -> None:
```

Comment:
clear and delete are synonyms.

how about `clear_key` and `clear_all` or something

Reply:
changed to `clear_key` and `clear_all`